modules/mobile/docs/openapi.json

```diff
@@ -23759,6 +23806,37 @@
             }
           }
         }
+      },
+      "VetVerificationStatus": {
+        "type": "object",
+        "additionalProperties": false,
+        "required": [
+          "data"
+        ],
+        "properties": {
+          "data": {
+            "type": "object",
+            "additionalProperties": false,
+            "required": [
+              "attributes"
+            ],
+            "properties": {
+              "attributes": {
+                "type": "object",
+                "additionalProperties": true,
+                "required": [
+                  "veteranStatus"
+                ],
+                "properties": {
+                  "veteranStatus": {
```

Comment:
Similar question as the schema do we need to include anything about the `notConfirmedReason`